src/dcd_mapping/cli.py

```diff
@@ -25,6 +25,14 @@
     default=False,
     help="Enable debug logging",
 )
+@click.option(
+    "--hide",
```

Comment:
Generally you're supposed to name this "quiet" in CLI tools (https://www.gnu.org/prep/standards/standards.html#Option-Table)